src/services/services.ts

```diff
@@ -248,6 +159,74 @@ namespace ts {
         }
     }
 
+    function createChildren(node: Node, sourceFile: SourceFileLike = node.getSourceFile()): Node[] {

```

Comment:
no need to do this initialization all the time. the idea was to defer the walk up the tree until you need the source file. e.g. no children, or in jsDoc